src/common/Executor.cpp

```diff
@@ -919,22 +968,28 @@ void Executor::runKernel(KernelBase* kernel, bool print_kernel_name)
 
     if ( run_params.showProgress() ) {
       if ( kernel->hasVariantDefined(vid) ) {
-        getCout() << "   Running ";
+        getCout() << "\tRunning ";
       } else {
-        getCout() << "   No ";
+        getCout() << "\tNo ";
       }
       getCout() << getVariantName(vid) << " variant" << endl;
     }
 
     for (size_t tune_idx = 0; tune_idx < kernel->getNumVariantTunings(vid); ++tune_idx) {
-
-      if ( run_params.showProgress() ) {
-        getCout() << "     Running "
-                  << kernel->getVariantTuningName(vid, tune_idx) << " tuning";
+      std::string tuning_name = kernel->getVariantTuningName(vid, tune_idx);
+      if (tuning_names_order_map.find(tuning_name) != tuning_names_order_map.end()) { // Check if valid tuning
+        if ( run_params.showProgress() ) {
+          getCout() << "\t\tRunning "
+                    << tuning_name << " tuning";
+        }
+        kernel->execute(vid, tune_idx); // Execute kernel
+        if ( run_params.showProgress() ) {
+          getCout() << " -- " << kernel->getLastTime() << " sec." << endl;
+        }
       }
-      kernel->execute(vid, tune_idx);
-      if ( run_params.showProgress() ) {
-        getCout() << " -- " << kernel->getLastTime() << " sec." << endl;
+      else {
+        getCout() << "\t\tNo "
```

Comment:
```suggestion
        getCout() << "\t\tSkipping "
```

Reply:
Variant also has this, but it's a different meaning since that variant may not exist for the kernel, right?
```
Run kernel -- Algorithm_SORT
        No Base_CUDA variant
```
Whereas we do not print the tunings if they do not exist for the kernel, so "skipping" is appropriate.
```
Run kernel -- Algorithm_REDUCE_SUM
        Running Base_CUDA variant
                No cub tuning
                Running block_128 tuning -- 0.00268784 sec.
                No block_256 tuning
                No block_512 tuning
                No block_1024 tuning

Run kernel -- Algorithm_MEMSET
        Running Base_CUDA variant
                No library tuning
                Running block_128 tuning -- 0.00148195 sec.
                No block_256 tuning
                No block_512 tuning
                No block_1024 tuning
```

Reply:
Yep.